src/client/src/orangehrmRecruitmentPlugin/pages/public-job-vacancy.scss

```diff
@@ -0,0 +1,183 @@
+// if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301, USA
+
+@import '@ohrm/oxd/styles/_mixins.scss';
+@import '@/core/styles/mixins.scss';
+
+.orangehrm {
+  &-background-container {
+    height: 80%;
+  }
+
+  &-applicant {
+    &-card {
+      &-pre-tag {
+        margin-top: 0;
+        margin-bottom: 0;
+        font-family: $oxd-font-family;
+        white-space: pre-wrap;
+        white-space: -moz-pre-wrap;
+        white-space: -o-pre-wrap;
+        word-wrap: break-word;
+      }
+
+      &-footer {
+        display: flex;
+        justify-content: left;
+      }
+
+      &-anchor-tag {
+        color: $oxd-primary-one-color;
+        padding-top: 0.5rem;
+      }
+
+      &-anchor-tag:hover {
+        cursor: pointer;
+      }
+    }
+
+    &-container {
+      width: 100%;
+      display: block;
+
+      &-row {
+        width: inherit;
+      }
+
+      &-colspan-2 {
+        grid-column: 1 / span 2;
+      }
+
+      &-grid-checkbox {
+        .oxd-input-group {
+          flex-direction: row-reverse;
+          justify-content: flex-end;
+        }
+      }
+    }
+  }
+
+  &-container {
+    height: 100%;
+    @include oxd-scrollbar();
+    overflow: auto;
+    position: relative;
+    margin: 0;
+
+    &-loader {
+      margin: 0 auto;
+      background-color: $oxd-white-color;
+      position: absolute;
+      top: 50%;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    }
+
+    &-img {
+      height: 40px;
+    }
+  }
+
+  &-card-container {
+    width: 80%;
+    margin: 1rem auto;
+    padding: 0.75rem 1.5rem;
+
+    &-header {
+      display: flex;
+      justify-content: space-between;
+      flex-direction: column-reverse;;
+      @include oxd-respond-to('md') {
+        flex-direction: row;
+      }
+    }
+  }
+
+  &-paper-container {
+    width: 80%;
+    margin: 1rem auto;
+    padding: 0.75rem 1.5rem;
+  }
+
+  &-vacancy {
+    &-card {
+      display: block;
+
+      &-header {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      &-anchor-tag {
+        color: $oxd-primary-one-color;
+        padding-top: 0.5rem;
+      }
+
+      &-anchor-tag:hover {
+        cursor: pointer;
+      }
+
+      &-body {
+        @include truncate(4,15px,#ffff );
+        @include oxd-respond-to('md') {
+          @include truncate(2,15px,#ffff );
+        }
+
+      }
+
+      &-pre-tag {
+        margin-top: 0;
+        font-family: $oxd-font-family;
+        white-space: pre-wrap;
+        white-space: -moz-pre-wrap;
+        white-space: -o-pre-wrap;
+        word-wrap: break-word;
+      }
+
+      &-footer {
+        display: flex;
+        justify-content: left;
+      }
+    }
+
+    &-list-footer {
+      padding-top: 8px;
+      padding-left: 40px;
+      @include oxd-respond-to('md') {
+        margin-bottom: -32px;
+        padding-left: 120px;
+      }
+      @include oxd-respond-to('xl') {
```

Comment:
if it applies for `md`, then it will also apply for `xl`. no need for repeating the same

Reply:
done